test/lib/OAuth2/Storage/Bootstrap.php

```diff
@@ -1,888 +1,996 @@
+        $pdo->prepare($sql)->execute(array('oauth_test_client', 'test_subject', $this->getTestPublicKey()));
+    }
+
+    public function getSqliteDir()
+    {
+        return $this->configDir. '/test.sqlite';
+    }
+
+    public function getConfigDir()
+    {
+        return $this->configDir;
+    }
+
+    private function createCouchbaseDB(\Couchbase $db)
+    {
+        $db->set('oauth_clients-oauth_test_client',json_encode(array(
+            'client_id' => "oauth_test_client",
+            'client_secret' => "testpass",
+            'redirect_uri' => "http://example.com",
+            'grant_types' => 'implicit password'
+        )));
+
+        $db->set('oauth_access_tokens-testtoken',json_encode(array(
+            'access_token' => "testtoken",
+            'client_id' => "Some Client"
+        )));
+
+        $db->set('oauth_authorization_codes-testcode',json_encode(array(
+            'access_token' => "testcode",
+            'client_id' => "Some Client"
+        )));
+
+        $db->set('oauth_users-testuser',json_encode(array(
+            'username' => 'testuser',
+            'password' => 'password',
+            'email' => 'testuser@test.com',
+            'email_verified' => true,
+        )));
+
+        $db->set('oauth_jwt-oauth_test_client',json_encode(array(
+            'client_id' => 'oauth_test_client',
+            'key'       => $this->getTestPublicKey(),
+            'subject'   => 'test_subject',
+        )));
+    }
+
+    private function clearCouchbase(\Couchbase $cb)
+    {
+        $cb->delete('oauth_authorization_codes-new-openid-code');
+        $cb->delete('oauth_access_tokens-newtoken');
+        $cb->delete('oauth_authorization_codes-newcode');
+        $cb->delete('oauth_refresh_tokens-refreshtoken');
+    }
+
+    private function createMongo(\MongoDB $db)
+    {
+        $db->oauth_clients->insert(array(
+            'client_id' => "oauth_test_client",
+            'client_secret' => "testpass",
+            'redirect_uri' => "http://example.com",
+            'grant_types' => 'implicit password'
+        ));
+
+        $db->oauth_access_tokens->insert(array(
+            'access_token' => "testtoken",
+            'client_id' => "Some Client"
+        ));
+
+        $db->oauth_authorization_codes->insert(array(
+            'authorization_code' => "testcode",
+            'client_id' => "Some Client"
+        ));
+
+        $db->oauth_users->insert(array(
+            'username' => 'testuser',
+            'password' => 'password',
+            'email' => 'testuser@test.com',
+            'email_verified' => true,
+        ));
+
+        $db->oauth_jwt->insert(array(
+            'client_id' => 'oauth_test_client',
+            'key'       => $this->getTestPublicKey(),
+            'subject'   => 'test_subject',
+        ));
+    }
+    
+    private function createMongoDB()
+    {
+        $bulk = new \MongoDB\Driver\BulkWrite();
+        $bulk->insert([

```

Comment:
we need to use `array()` syntax here, since this library is compatible with PHP 5.3